File: app/src/main/java/com/example/projeto_ds2/MainActivity.java
package com.example.projeto_ds2;

import androidx.appcompat.app.AppCompatActivity;
import androidx.appcompat.app.AppCompatDelegate;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        AppCompatDelegate.setDefaultNightMode(AppCompatDelegate.MODE_NIGHT_NO);
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Button b = findViewById(R.id.button);

        b.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {

                Intent intent = new Intent(MainActivity.this, MensagensActivity.class);
                startActivity(intent);

            }
        });

        Button buttonAvisos = findViewById(R.id.button2);

        buttonAvisos.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {

                Intent intent = new Intent(MainActivity.this, AvisoActivity.class);
                startActivity(intent);

            }
        });

    }
}

File: app/src/main/java/br/edu/up/vivianmarcal/MainActivity.kt
package br.edu.up.vivianmarcal

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.appcompat.app.AppCompatDelegate
import com.example.projeto_ds2.R
import android.content.Intent
import android.util.Log
import android.widget.Button
import android.widget.ToggleButton
import br.edu.up.vivianmarcal.model.usuario.TipoUsuario
import br.edu.up.vivianmarcal.model.usuario.Usuario

class MainActivity : AppCompatActivity() {

    private var usuario: Usuario? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        AppCompatDelegate.setDefaultNightMode(AppCompatDelegate.MODE_NIGHT_NO)
        super.onCreate(savedInstanceState)

        if (usuario == null){
            Log.v("App", "LOG: setando usuario como PAI")
            usuario = Usuario(5678.toLong(), "nomeQualquer", TipoUsuario.PAI_ALUNO)
        }

        setContentView(R.layout.activity_main)
        val b = findViewById<Button>(R.id.button)
        b.setOnClickListener {
            val intent = Intent(this@MainActivity, MensagensActivity::class.java)
            intent.putExtra("Usuario", usuario)
            startActivity(intent)
        }
        val buttonAvisos = findViewById<Button>(R.id.button2)
        buttonAvisos.setOnClickListener {
            val intent = Intent(this@MainActivity, AvisoActivity::class.java)
            intent.putExtra("Usuario", usuario)
            startActivity(intent)
        }

        val buttonTrocaUsuario = findViewById<ToggleButton>(R.id.trocaUsuario)
        buttonTrocaUsuario.setOnCheckedChangeListener { _, isChecked ->
            try {
                if (isChecked){
                    Log.v("App", "LOG: isChecked")

                    usuario!!.id = 1234.toLong()
                    usuario!!.tipoUsuario = TipoUsuario.ESCOLA
                }else{
                    Log.v("App", "LOG: isNotChecked")

                    usuario!!.id = 5678.toLong()
                    usuario!!.tipoUsuario = TipoUsuario.PAI_ALUNO
                }
            } catch (e: Exception) {
                Log.v("App", "ERRO: " + e.message)
            }
        }

    }
}